app/models/incoming_message.rb

```diff
@@ -55,7 +55,8 @@ class IncomingMessage < ActiveRecord::Base
   # never really has many info_request_events, but could in theory
   has_many :info_request_events, :dependent => :destroy
 
-  belongs_to :raw_email, :dependent => :destroy
```

Comment:
Links / references to what says not to do this?

http://guides.rubyonrails.org/v4.1.16/association_basics.html#options-for-belongs-to-dependent suggests you can?